projects/ama-sdk/src/lib/variables/variables.component.html

```diff
@@ -1,32 +1,15 @@
 <h2 mat-dialog-title><mat-icon class="variables-icon">layers</mat-icon>{{title | translate}}</h2>
 
 <mat-dialog-content class="properties-content variables-dialog-content" data-automation-id="variables-dialog">
-    <mat-tab-group [@.disabled]="true" selectedIndex="0">
-
-        <mat-tab label="{{ 'SDK.VARIABLES_EDITOR.TABS.VIEWER' | translate }}">
```

Comment:
Please check if we can remove either the translation keys or not. If they are not used anywhere else, please get rid of them as well.